### PR TITLE
Update cargo-embed.md

### PR DESCRIPTION
**Pull request by dcarosone**
Add some more notes to explain the two-level configuration-name structure

### CODE REVIEW

File: content/guide/1_tools/cargo-embed.md [hunk 1]
```diff
@@ -214,4 +217,4 @@ automatically be displayed for channels with a corresponding down channel. This
 must be the same for the up and down channels. This is the default rttui behavior and can be configured. RTT itself can handle any up/down
 numbers combination.<br>
 <br>
-Now you should be able to debug conveninently. Happy coding!
\ No newline at end of file
+Now you should be able to debug conveninently. Happy coding!
```
Comment (Yatekii) [inline content/guide/1_tools/cargo-embed.md:220]:
```suggestion
Now you should be able to debug conveniently. Happy coding!
```

**Review by Yatekii — APPROVED** [review]
LGTM, thank you!

I fixed a typo of mine, I hope thats ok :>

**Comment by Yatekii** [comment]
Could you rebase this onto master where @Tiwalun fixed this? :)

**Comment by dcarosone** [comment]
I don't know why CI is failing, it tells me it can't currently download the log.

**Comment by Yatekii** [comment]
It has something to do with the GIthub action we use for Zola deployment. I thought we fixed it but somehow it still fails :/

**Comment by Tiwalun** [comment]
It might be a problem with the permissions, in which case it would fail when the action is run on a forked branch.

**Pull request merged**